### PR TITLE
Update mappings

### PR DESCRIPTION
**Pull request by chrzyki**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

We should probably refine more of the WALL mappings before release. I think most of the remaining ones (except of course for the very explicit WALL OF HOUSE glosses) should be changed to WALL. What do you think, @LinguList?

### CODE REVIEW

File: concepticondata/conceptlists/Dawson-1887-731.tsv [hunk 1]
```diff
@@ -157,7 +157,7 @@ Dawson-1887-731-155	155	Mat	195	MAT	Dwellings	4	29
 Dawson-1887-731-156	156	Bed	1663	BED	Dwellings	4	29
 Dawson-1887-731-157	157	Floor	1632	FLOOR	Dwellings	4	29
 Dawson-1887-731-158	158	Ceiling			Dwellings	4	29
-Dawson-1887-731-159	159	Wall	933	WALL (OF HOUSE)	Dwellings	4	29
+Dawson-1887-731-159	159	Wall	3830	WALL	Dwellings	4	29
```
Comment (CarolinHu) [inline concepticondata/conceptlists/Dawson-1887-731.tsv:160]: Could this be indeed WALL (OF HOUSE) as opposed to the ceiling above?

**Review by CarolinHu — APPROVED** [review]
I can't speak for the Chinese mappings, but the rest looks good.
Regarding your proposal to remap almost all of them, does it make sense to add a relation WALL narrower WALL (OF HOUSE)?

**Comment by chrzyki** [comment]
> I can't speak for the Chinese mappings, but the rest looks good.
> Regarding your proposal to remap almost all of them, does it make sense to add a relation WALL narrower WALL (OF HOUSE)?

Sense relations in general would be good to have for the new concepts. Do you maybe have time to look at this?

File: concepticondata/conceptlists/Dawson-1887-731.tsv (hunk 1 above)
Comment (chrzyki) [reply concepticondata/conceptlists/Dawson-1887-731.tsv:160]: In this instance in can indeed be both (just going by the lexemes), so the more general mapping is okay I'd say.

**Comment by CarolinHu** [comment]
Yes, I'll have a look.

**Pull request merged**